routers/api/v1/api.go

```diff
@@ -788,6 +790,13 @@ func verifyAuthWithOptions(options *common.VerifyOptions) func(ctx *context.APIC
 	}
 }
 
+// check for and warn against deprecated authentication options
+func checkDeprecatedAuthMethods(ctx *context.APIContext) {
+	if ctx.FormString("token") != "" || ctx.FormString("access_token") != "" {
+		ctx.Resp.Header().Set("Warning", "token and access_token API authentication is deprecated")
```

Comment:
```suggestion
		ctx.Resp.Header().Set("Warning", "token and access_token API authentication is deprecated and will be remove in gitea 1.23. Please use AuthorizationHeaderToken instead.")
```

Reply:
Thanks. A tiny nit myself, could you clarify that it is the query string portion that is being deprecated as tokens would remain functional when being passed as headers.